LoginDemo/LearningSelenium/FactoryDesign/FactoryMethodDesign/FMDTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿using LearningSelenium.Utilities;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace LearningSelenium.FactoryDesign.FactoryMethodDesign
+{
+    public class FMDTest
+    {
+        WebDriver driver;
+        Utility utilityObj;
+        ReadConfig config = ReadConfig.GetReadConfig("C:/Kanaga/repo/Login/LoginDemo/LoginDemo/LearningSelenium/FactoryDesign/DataConfig.json");
```

Comment:
Read about relative and absolute paths. We usually use relative paths

Reply:
relative path updated as @"..\..\..\DataConfig.json"